src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  height: 408px;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  transition: transform 300ms;
+
+  &__image {
```

Comment:
The BEM element '__image' should be nested within its block '.card' using '&' to concatenate parent selector name, which is a common practice in BEM with SCSS.